Azure/Azure_VM/VM_Creat.md

```diff
@@ -0,0 +1,15 @@
+## About Azure 
+###  1. Certification Road Map
+#### ref : https://www.whizlabs.com/blog/azure-certifications-path/
+### 2. Azure Free Account Creat link
+#### https://azure.microsoft.com/en-in/free/
+###  3. Create a Windows virtual machine in the Azure portal 
+####  ref : https://docs.microsoft.com/en-us/azure/virtual-machines/windows/quick-create-portal
+### 4. Create a Windows virtual machine in Azure with PowerShell
+#### ref : https://docs.microsoft.com/en-us/azure/virtual-machines/windows/quick-create-powershell
+### 5. Create a public IP address using the Azure portal
+#### ref : https://docs.microsoft.com/en-us/azure/virtual-network/ip-services/create-public-ip-portal?tabs=option-1-create-public-ip-standard
+### 6. Create a public IP address using PowerShell
+#### ref : https://docs.microsoft.com/en-us/azure/virtual-network/ip-services/create-public-ip-powershell?tabs=create-public-ip-standard%2Ccreate-public-ip-zonal%2Crouting-preference
+### 7. Storage Pricing for Azure virtual machines
+#### ref : https://azure.microsoft.com/en-us/pricing/details/managed-disks/#pricing
```

Comment:
macの場合、mstsc /v:publicIpAddress が実行できなかったので、microsoft remote desktop をDLした気がするので、以下のNoteをどこかに記入していただけますでしょうか。

```
for mac, instead of running command on mac, mstsc /v:publicIpAddress, 
do as follow:
 
- download 「microsoft remote desktop」 from apple store
- open app & create desktop 
- add public IP address, username & password
```